frontend/src/components/design-system/containers/PillComponent.vue

```diff
@@ -1,17 +1,43 @@
 <script setup lang="ts">
+import { computed } from 'vue';
 import LabelText from '@/components/design-system/typography/LabelText.vue';
 
 export interface Props {
   active?: boolean;
 }
 
-withDefaults(defineProps<Props>(), {
+const classes = computed(() => ({
+  'pill--active': props.active,
+}));
+
+const props = withDefaults(defineProps<Props>(), {
   active: false,
 });
 </script>
 
 <template>
-  <LabelText>
-    <slot />
-  </LabelText>
+  <div :class="['pill', classes]">
+    <LabelText>
+      <slot />
+    </LabelText>
+  </div>
 </template>
+
+<style lang="scss" scoped>
+.pill {
+  border-color: rgba(var(--color-neutral-border), 1);
+  border-radius: var(--radius-medium);
+  border-style: solid;
+  border-width: var(--spacing-pico);
+  background-color: rgb(var(--color-neutral-bg));
+  color: rgb(var(--color-neutral-fg));
+  padding-inline: var(--spacing-small);
+  padding-block: var(--spacing-extra-small);
+
+  &--active {
+    border-color: rgb(var(--color-neutral-border-strong), 1);
```

Comment:
Naštěstí CSS je dost tolerantní... jazyk?, takže rgb s alpha kanálem nic nerozbije a dokonce to funguje. Ale lepší je to nepokoušet a u `rgb` neuvádět alphu.

Reply:
Upraveno.